README.md

```diff
@@ -39,6 +39,25 @@ PRIMARY_LANG=en
 
 Once you saved the .env, the bot should be good to go. Simply run it by `npm start`.
 
+## Docker container
+
+To run as a docker container run:
+
+```
+docker run \
+  --name "/TwitchChatTranslator" \
+  --runtime "runc" \
+  --env "BOT_USERNAME=YourBotUserName" \
+  --env "TWITCH_OAUTH=YourBotsOAuthKey" \
+  --env "CHANNEL_NAME=YourTwitchChannel" \
+  --env "AZURE_SUB_KEY=YourAzureSubscriptionKey" \
+  --env "PRIMARY_LANG=en" \
+  --detach \
+  --entrypoint "docker-entrypoint.sh" \
+  "scsa20/twitch-chat-translator-bot:latest" \
```

Comment:
Almost perfect, just need to edit this so it's `faizal101/twitch-chat-translator-bot:latest`

Reply:
But you don't even have a docker hub

https://l.sc20.me/xdbp0

Whereas I do have it up on docker thus why I have it the way it is, if you want it to be your account then please upload the image to your docker hub OR to your github (which is another thing you do not have as well).